### ISSUE TITLE
Wrong EBIT numbers with MSFT example

### ISSUE THREAD

**Issue by byagiz** [ranaroussi/yfinance#844]
Hi guys,

I was playing around with the examples in the code description page and realised that:

> import yfinance as yf
> MSFT = yf.Ticker("MSFT")
> print(MSFT.financials)

shows wrong EBIT numbers for MSFT. I read from my terminal following output:
2021-06-30         2020-06-30         2019-06-30        2018-06-30
69916000000.0   53145000000.0   42959000000.0   35058000000.0

cross checking with yahoo's website shows me different numbers:
        TTM                 6/29/2021        6/29/2020       6/29/2019       6/29/2018
EBIT	73,448,000	73,448,000	55,627,000	46,374,000	39,207,000

Is there a reason why this is happening? If you have any questions please contact me!

**Comment by asafravid** [comment]
Seems you’re trying to compare yearly values to quarterly values… please recheck

Asaf - אסף

**Comment by walterlsb** [comment]
I'm getting a problem with ebit figures not matching as well.
This is my code and result:


```
import yfinance as yf
MSFT = yf.Ticker("MSFT")
fin = MSFT.get_financials(as_dict=True, freq='quarterly')
ebitTTM = 0
for key in fin.keys():
    print(key,fin[key]['Ebit'])
    ebitTTM = ebitTTM + fin[key]['Ebit']

print('TTM',ebitTTM)

```
Result:
```
2021-09-30 00:00:00 20238000000.0
2021-06-30 00:00:00 19095000000.0
2021-03-31 00:00:00 17048000000.0
2020-12-31 00:00:00 17897000000.0
TTM 74278000000.0
```

The website shows:
EBIT (x1000)

TTM: 77,798,000
9/30/2021: 21,063,000
6/30/2021: 19,958,000
3/31/2021: 17,869,000
12/31/2020: 18,908,000

I can see the numbers are similar - but shouldn't they be the same?

**Comment by teeejer** [comment]
When I check for EBIT values, it looks like I'm actually getting the Operating Income values.
```
Ebit                                     69916000000.0   53145000000.0   42959000000.0   35058000000.0
Operating Income                         69916000000.0   53145000000.0   42959000000.0   35058000000.0
```
This is my workaround for what it's worth.
```
MSFT = yf.Ticker("MSFT")
fin = MSFT.get_financials(as_dict=True, freq='yearly')
ebit = 0
for key in fin.keys():
    ebit = fin[key]['Income Before Tax'] - fin[key]['Interest Expense']
    print(ebit)
```

**Comment by ValueRaider** [comment]
Fixed in `yfinance` 0.2
